modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -132,7 +133,7 @@ static GDScriptParser::DataType make_builtin_meta_type(Variant::Type p_type) {
 	return type;
 }
 
-bool GDScriptAnalyzer::has_member_name_conflict_in_script_class(const StringName &p_member_name, const GDScriptParser::ClassNode *p_class) {
+bool GDScriptAnalyzer::has_member_name_conflict_in_script_class(const StringName &p_member_name, const GDScriptParser::ClassNode *p_class, bool p_is_func) {
```

Comment:
I'm not sure about the added parameter type. I would pass the type and let the method do the comparison instead of passing a highly specific boolean that is used once.

Reply:
Haha, I was doing that when I thought `has_member_name_conflict_in_script_class` needed to check for overloaded function signatures, but then reverted to something simpler when that turned out to already be done elsewhere.

But I agree. It is more flexible that way!

Reply:
Done!